server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts

```diff
@@ -185,8 +185,10 @@ export class DocumentPartition {
 		}
 	}
 
-	private updateActivityTime() {
-		this.activityTimeoutTime =
+	private updateActivityTime(activityTime?: number) {
+		const cacluatedActivityTimeout =
 			Date.now() + (this.lambda?.activityTimeout ?? this.activityTimeout);
+		this.activityTimeoutTime =
```

Comment:
Isn't this equivalent to:

`this.activityTimeoutTime = activityTime ?? calculatedActivityTimeout;`

Reply:
I used this before, but it will give unexpected behavior if the activityTime is 0 (no assignment in this case). We are unlikely to pass this value though.